c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -545,6 +545,13 @@ impl TypedAstContext {
                     to_walk.push(decl_id);
                     used.insert(decl_id);
                 }
+                CDeclKind::Function {
+                    body: Some(_),
+                    ..
+                } if preserve_unused_functions => {
+                    to_walk.push(decl_id);
+                    used.insert(decl_id);
```

Comment:
The `used` variable no longer tracks what is used because of the option to preserve unused functions. Now it tracks what we want to keep, so we should consider renaming it to `keep` or similar to reflect that.

Reply:
Happy to adjust that, though when I get to that level I'll also want to refactor the whole match.

Reply:
Renamed in b6260dbdf94b8c0009db4f7dcae2f23c31de95cd now; I'm not sure if you'll be happy with it because doing it all the way through meant renaming the function as well (for otherwise things would be inconsistent there). Just have a look, I'd understand if you rather not have that commit.